ui/job-view/PushJobs.jsx

```diff
@@ -32,15 +63,11 @@ export default class PushJobs extends React.Component {
     this.filterPlatformCallback = this.filterPlatformCallback.bind(this);
 
     this.state = {
-      platforms: {},
+      platforms: [],
```

Comment:
``platforms`` didn't need to be an object.  It made the code a little messier having it that way, so we store it here as just an array now.  Sorry this adds a few more changes to the code.